src/components/ProjectPost.js

```diff
@@ -228,18 +224,21 @@ const UndecoratedProjectPostCard = ({ post, community, comments, dispatch, isMob
     </div>}
     <Supporters post={post} simple />
     <div className='comments-section-spacer' />
-    {canComment(currentUser, post) && <CommentSection post={post}
-      comments={comments}
-      onExpand={() => dispatch(navigate(url))} />}
+    <CommentSection
+      onExpand={() => dispatch(navigate(url))}
+      isProjectRequest
+      {...{post, canComment, comments}}
+    />
```

Comment:
isProjectRequest is a bool prop and there is no corresponding value set. I could put it in the expanded props list below but I'd have to do isProjectRequest: true and it feels less better at least to me in this case :). . . Changing formatting slightly either way. I think you'll see what I mean.